KEYCLOAK-15244 Require user to agree to 'terms and conditions' during registration (#7938)
keycloak/keycloak: docs/documentation/server_admin/topics/users/proc-requiring-tac-agreement-at-registration.adoc

// Module included in the following assemblies:
//
// con-user-registration.adoc

[id="proc-requiring-tac-agreement-at-registration_{context}"]
= Requiring user to agree to terms and conditions during registration

[role="_abstract"]

> You can condition a user registration to his agreement to your terms and conditions.

Reviewer: ```suggestion
For a user to register, you can require agreement to your terms and conditions.
```

.Registration form with required terms and conditions agreement
image:images/registration-form-with-required-tac.png[]

.Prerequisite
* User registration is enabled.

> * Terms and conditions required action is enabled

Reviewer: ```suggestion
* Terms and conditions required action is enabled.
```
